cmd/manager/main.go

```diff
@@ -59,26 +59,45 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+type stringList []string
+
+func (f *stringList) String() string {
+	return "multiple values"
+}
+
+func (s *stringList) Set(value string) error {
+	values := strings.Split(value, ",")
+	*s = append(*s, values...)
+	return nil
+}
+
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var openshift bool
 	var probeAddr string
+	var additionalNamespaces stringList
 
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 
+	flag.StringVar(&controllers.KeplerDeploymentNS, "deployment-namespace", controllers.KeplerDeploymentNS,
+		"Namespace where kepler and its components are deployed.")
+
+	flag.CommandLine.Var(flag.Value(&additionalNamespaces), "watch-namespaces",
```

Comment:
can we elaborate a bit how this parameter is to be used? is this for running multiple kepler-exporters?

Reply:
I am also thinking of allowing a `*` as one of the options to allow watching all namespaces but lets do it when we really find a need.